tests/conftest.py

```diff
@@ -60,7 +60,7 @@ def generate_tile(width=32, height=32, blocks=50):
     """Generate a widthXheightX3 image, with blocks pixels surrounded by ones and the rest zeros in band 0"""
     image = np.zeros((width, height, 1), np.float32)
     label = np.zeros((width, height), np.uint8)
-    for _ in range(blocks):
+    for _ in np.arange(blocks):
```

Comment:
Since this is just the test suite it doesn't matter to me which range function is used, but is there any specific reason you prefer np.arange as opposed to range for this for loop?

Reply:
yeah you are right, np.arange dont matter here. but I just added np.arange performance, because it's faster than python range function.